admin/app/adapters/certification.js

```diff
@@ -2,15 +2,15 @@ import ApplicationAdapter from './application';
 
 export default ApplicationAdapter.extend({
   urlForFindRecord(id) {
-    return `${this.host}/admin/certifications/${id}`;
+    return `${this.host}/${this.namespace}/admin/certifications/${id}`;
   },
 
   urlForUpdateMarks() {
-    return `${this.host}/admin/assessment-results/`;
+    return `${this.host}/${this.namespace}/admin/assessment-results/`;
   },
 
   urlForUpdateRecord(id) {
-    return `${this.host}/certification-courses/${id}`;
+    return `${this.host}/${this.namespace}/certification-courses/${id}`;
   },
```

Comment:
Attention, on est pas `iso` là ? Y'a pas de `/admin` côté api ?

Reply:
_Malheureusement_, non.

Reply:
Donc faut pas le `this.namespace` à cet endroit

Reply:
@Radioreve pourquoi ? C'est `api` le namespace pas `admin` !?

Reply:
Au sein de **certification-course-controller** (côté API), toutes les routes contiennent "/admin" sauf la dernière, `PATCH /api/certification-courses/{id}`

Le code correspondant côté front est le suivant :

```# /app/adapters/certification.js

urlForUpdateRecord(id) {
  return `${this.host}/${this.namespace}/certification-courses/${id}`;
},```

Dans le store Ember, la méthode `urlForUpdateRecord` provoque une **requête PATCH** (la fameuse).

Pour reproduire, il faut ouvrir l’onglet “Informations” d’une certification, et par exemple, modifier son statut puis publier (et confirmer).

Moralité : le code qui a été poussé fonctionne !